tests/saitests/py3/switch.py

```diff
@@ -822,6 +822,33 @@ def sai_thrift_read_port_counters(client, asic_type, port):
     return (counters_results, queue_counters_results)
 
 
+def sai_thrift_get_voq_port_id(client, system_port_id):
+    object_id = client.sai_thrift_get_sys_port_obj_id_by_port_id(system_port_id)
+    voq_list = []
+    port_attr_list = client.sai_thrift_get_system_port_attribute(object_id)
+    attr_list = port_attr_list.attr_list
+    for attribute in attr_list:
+        if attribute.id == SAI_SYSTEM_PORT_ATTR_QOS_VOQ_LIST:
+            for voq_id in attribute.value.objlist.object_id_list:
+                voq_list.append(voq_id)
+    return (voq_list)
+
+
+def sai_thrift_read_port_voq_counters(client, voq_list):
+    cnt_ids = []
+    thrift_results = []
+    voq_counters_results = []
+    cnt_ids.append(SAI_QUEUE_STAT_PACKETS)
+    queue1 = 0
+    for voq in voq_list:
+        if queue1 <= 7:
```

Comment:
Why do we need queue1 here ? Can't we work with VoQ id directly ?

Reply:
Changed the naming convention to 'counter'. It is just a counter value.